tests/driver_pir/Makefile

```diff
@@ -11,20 +11,18 @@ USEMODULE += xtimer
 ifneq (,$(filter stm32f4discovery,$(BOARD)))
```

Comment:
I would completely remove the lines from here to the second endif

Reply:
I will do that if it has no negative side effect. I personally don't use these two boards.

Reply:
i don't know why there are specific case for these boards. But there's no PIR on them. And if it was really the case, then the defines should in the board.h file of each board.

Reply:
I agree. Let's just remove them :